wechat/render.py

```diff
@@ -153,6 +154,12 @@ def fallback():
         elif msg.type == TYPE_WX_VIDEO:
             # TODO: fetch video from resource
             return fallback()
+        elif msg.type == TYPE_VIDEO_FILE:
+            # TODO: fetch video from resource
```

Comment:
remove this comment

Reply:
Ok. Shall we put a TODO somewhere? As you may try out, video rendering is not so well, as I just did a trick letting fancybox dealing them as img.